index.d.ts

```diff
@@ -80,7 +79,8 @@ export function wrapStore<S>(
     portName: string,
     dispatchResponder?(dispatchResult: any, send: (response: any) => void): void,
     serializer?: Function,
-    deserializer?: Function
+    deserializer?: Function,
+    diffStrategy?: Function
```

Comment:
Seems like we should assign this to the `DiffStrategy` type, correct?

Reply:
I see. You're right.